packaging/firehol/firehol.functions

```diff
@@ -11,12 +11,12 @@ firehol_check_file() {
     sbin/Makefile.in)
       :
     ;;
-    configure.ac|sbin/commands.sed.in)
-      check_commands sbin/firehol.in || status=1
-      check_commands sbin/fireqos.in || status=1
-      check_commands sbin/link-balancer.in || status=1
-      check_commands sbin/update-ipsets.in || status=1
-      check_commands sbin/vnetbuild.in || status=1
+    configure.ac)
+      check_commands sbin/firehol.in.in || status=1
+      check_commands sbin/fireqos.in.in || status=1
+      check_commands sbin/link-balancer.in.in || status=1
+      check_commands sbin/update-ipsets.in.in || status=1
+      check_commands sbin/vnetbuild.in.in || status=1
```

Comment:
I do not understand when this is helpful, as something should be terribly wrong for this to alert anything, you probably want to move all these values into firehol.config and check only one file.